src/App.jsx

```diff
@@ -0,0 +1,20 @@
+import './App.css'
+import { Route, Routes } from 'react-router-dom'
+import Bookstate from './components/bookstate'
+import Nav from './components/Navigation'
+
+function App() {
+
+  return (
+    <>
+    <h1>book store</h1>
+    <Nav/>
+    <Routes>
+     <Route path='/' element={<Bookstate/>}/>
+     <Route path='/catigories'/> 
```

Comment:
- [ ] Kindly make sure the categories component is rendered here on the URL path `categories` as you may notice nothing is being rendered.
```suggestion
     <Route path='/catigories' element={<Categories/>}/> 
```